docs/circuit-conversations-list.md

```diff
@@ -0,0 +1,74 @@
+# circuit-conversations-list
```

Comment:
Haven't review this. Please review it yourself given the comments I had in the chat README